sdk/translation/test-resources.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+      "metadata": {
+        "description": "The tenant ID to which the application and resources belong."
+      }
+    },
+    "testApplicationOid": {
+      "type": "string",
+      "defaultValue": "b3653439-8136-4cd5-aac3-2a9460871ca6",
+      "metadata": {
+        "description": "The client OID to grant access to test resources."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location of the resource. By default, this is the same as the resource group."
+      }
+    },
+	"endpointPrefix": {
+		"defaultValue": "api",
+		"type": "string"
+	},
+	"endpointSuffix": {
+		"defaultValue": ".cognitive.microsofttranslator.com",
+		"type": "string"
+	},
+	"customEndpointSuffix": {
+		"defaultValue": ".cognitiveservices.azure.com/translator/text/v3.0",
+		"type": "string"
+	}
+  },
+  "variables": {
+    "uniqueSubDomainName": "[format('{0}', parameters('baseName'))]",
+	"endpointValue": "[format('https://{0}{1}', parameters('endpointPrefix'), parameters('endpointSuffix'))]",
+    "customEndpointValue": "[format('https://{0}{1}', parameters('baseName'),parameters('customEndpointSuffix'))]",
+    "regionValue": "[format('{0}', parameters('location'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.CognitiveServices/accounts",
+      "apiVersion": "2017-04-18",
```

Comment:
nitpick: Could we use a more recent API version? For example: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/cognitivelanguage/test-resources.json#L66

Reply:
Updated